assets/src/onboarding-wizard/index.js

```diff
@@ -90,13 +95,24 @@ Providers.propTypes = {
 domReady( () => {
 	const root = document.getElementById( APP_ROOT_ID );
 
-	if ( root ) {
-		render(
-
-			<Providers>
-				<SetupWizard closeLink={ CLOSE_LINK } finishLink={ FINISH_LINK } appRoot={ root } />
-			</Providers>,
-			root,
-		);
+	if ( ! root ) {
+		return;
 	}
+
+	const errorHandler = ( error ) => {
```

Comment:
This is actually the [`ErrorEvent`](https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent), not the `error` object, so we could use a more appropriate name:

```suggestion
	const errorHandler = ( event ) => {
```